client/src/components/registration/registration-form/RegistrationForm.js

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+
+// Local Imports
+import RegistrationSerivce from 'services/RegistrationService';
+import FormInput from 'components/shared/forms/FormInput';
+import Message from 'models/messaging/Messages';
+
+/*
+import RegistrationForm from 'components/registration/registration-form/RegistrationForm';
+
+<RegistrationForm />
+*/
+
+// TO DO: style compenent with Martial UI
+// TO DO: integrate yup schema validation
+// TO DO: make this a sub-component and make a wrapper login page component that utilizes this on the page
+/**
+ * Registration Form component.
+ *
+ * Currently acting as the entire Registration Page Component.
+ */
+
```

Comment:
Comments should be directly above the code it's describing (or in line with in). As for the JSDoc here, it shouldn't have a space between it and the class header.